README.md

```diff
@@ -1,2 +1,41 @@
-# book-library
-Building a book library using Javascript
+# Book Library 
+
+Building a book library using HTML and Javascript
+
+![screenshot](./assets/screenshot.png)
+
+
+## Built With
+
+- HTML
+- Bootstrap
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository on your local machine
+- cd into the folder
+
```

Comment:
## Add more instructions in the README
- [ ] Kindly add a live link to your page.
- [ ] Do ensure you have more elaborate instructions on how well to run the application locally, thank you!